transports/rabbitmq/index.md

```diff
@@ -14,6 +14,8 @@ tags:
 
 Provides support for sending messages over [RabbitMQ](http://www.rabbitmq.com/) using the [RabbitMQ .NET Client](https://www.nuget.org/packages/RabbitMQ.Client/).
 
+WARNING: The transport is not compatible with versions of the RabbitMQ broker earlier than 3.4.0.
```

Comment:
The problem with that is that it's not explicit that 3.3 includes 3.3.1, 3.3.2, etc. It is implied, but I think some readers will miss the implication.